src/forwardmodels/Diffusion Approximation/DAcylinder_layered.jl

```diff
@@ -449,10 +453,18 @@ end
         tmp2 = D[k - 1] * α[k - 1] * n[k - 1]^2 * γN
         tmp3 = exp(-2 * α[k - 2] * l[k - 2])
 
-        βN  =  tmp1 * (1 + tmp3)
-        βN +=  tmp2 * (1 - tmp3)
-        γN  =  tmp1 * (1 - tmp3)
-        γN +=  tmp2 * (1 + tmp3)
+        # βN  =  tmp1 * (1 + tmp3)
+        # βN +=  tmp2 * (1 - tmp3)
+        # γN  =  tmp1 * (1 - tmp3)
+        # γN +=  tmp2 * (1 + tmp3)
+
+        # 2 ns optimization for median time of "fluence_DA_Nlay_cylinder_CW"
+        a  =  muladd(tmp1, tmp3, tmp1)
```

Comment:
Looking at this a bit more. The original has 4 multiplies, 4 adds, 2 subtracts. This version has 4 mul adds and 2 additions.... I'm wondering if we can decrease this a bit more because there are some repeat multiples....

```julia
a = tmp1 * tmp3
        c = tmp1 + tmp2
        b = tmp2 * tmp3
        d = a - b

        βN = c + d
        γN = c - d
```
This has 2 multiplies, 2 adds, and 2 subtracts.......

I'm wondering if this is faster?

Reply:
I'm looking at a few other parts of the code and it looks like rearranging in this form will be possible for all the cases. Now if it is faster we will have to benchmark to see (not sure how good the compiler is at optimizing those repeated multiplications).....